NineChronicles.Headless/GraphTypes/States/Models/Table/StakeRegularRewardInfoType.cs

```diff
@@ -17,7 +17,13 @@ public StakeRegularRewardInfoType()
             );
             Field<NonNullGraphType<StakeRewardEnumType>>(
                 nameof(StakeRegularRewardSheet.RewardInfo.Type),
-                resolve: context => context.Source.Type);
+                resolve: context => context.Source.Type
+            );
+            Field<StakeRewardEnumType>(
```

Comment:
Wait... Why is the `CurrencyTicker`'s type is `StakeRewardEnumType`? 🤔 It seems invalid type since `CurrencyTicker`'s type is `string`.

Reply:
And we also already have [CurrencyEnumType](https://github.com/planetarium/NineChronicles.Headless/blob/development/NineChronicles.Headless/GraphTypes/CurrencyEnumType.cs)

Reply:
practically, we're using both `CurrencyEnumType` and `string` like [this](https://github.com/planetarium/NineChronicles.Headless/blob/development/NineChronicles.Headless/GraphTypes/SimplifyCurrencyInputType.cs)

Reply:
It looks just a typo.😉

```suggestion
            Field<StringGraphType>(
```